lua/lspconfig.lua

```diff
@@ -7,7 +7,13 @@ local M = {
 M._root = {}
 
 function M.available_servers()
-  return vim.tbl_keys(configs)
+  local servers = {}
+  for server, config in pairs(configs) do
+    if config.manager ~= nil then
+      table.insert(servers, server)
```

Comment:
nit: I think vim.tbl_filter could save some code here

Reply:
It needs to access both the key and the value of the `configs` table, I assumed `tbl_filter` only passes a single argument (`value`) to the predicate function?

Reply:
oh... that's unfortunate.